docs/reference/migration/migrate_7_13.asciidoc

```diff
@@ -154,14 +154,15 @@ aggregations are rarely useful and often unintended.
 The `path.data` setting accepts a list of data paths, but if you specify
 multiple paths then the behaviour is unintuitive and usually does not give the
 desired outcomes. Support for multiple data paths is now deprecated and will be
-removed in 8.0.0.
+removed in a future release.
 
 *Impact* +
-Specify a single path in `path.data`. If needed, you can create a filesystem
-which spans multiple disks with a hardware virtualisation layer such as RAID,
-or a software virtualisation layer such as Logical Volume Manager (LVM) on
-Linux or Storage Spaces on Windows. If you wish to use multiple data paths on a
-single machine then you must run one node for each data path.
+To avoid deprecation warnings, specify a single path in `path.data`. If needed,
```

Comment:
I only added "To avoid deprecation warnings," and rewrapped. All other text is original.